src/chocolatey/infrastructure.app/commands/ChocolateyListCommand.cs

```diff
@@ -102,9 +104,18 @@ public virtual void configure_argument_parser(OptionSet optionSet, ChocolateyCon
                  .Add("id-starts-with",
                      "IdStartsWith - Only return packages where the id starts with the search filter. Available in 0.9.10+.",
                      option => configuration.ListCommand.IdStartsWith = option != null)
+                 .Add("o=|order=|order-by=",
+                     "OrderBy - Sort by package results by name (default), title, popularity, lastpublished. Available in 0.10.12+.",
+                     option => configuration.ListCommand.OrderBy = (PackageOrder)Enum.Parse(typeof(PackageOrder), option))

```

Comment:
This will throw an error. Please search and see how we parse enums in the code base. Then follow suit, thanks!

Reply:
If I pass an incorrect value

Reply:
I believe we do a TryParse and provide a default value. Apologies I didn't write more info here

Reply:
Ok thanks, will check

Reply:
Done